apps/dashboard-service/src/dashboard/dto/tree-restoration-goal.dto.ts

```diff
@@ -0,0 +1,74 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { populateDto } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+class TreeRestorationData {
+  @ApiProperty({
+    description: "Due date for this restoration data",
+    type: String,
+    example: "2024-01-01T00:00:00.000000Z"
+  })
+  dueDate: string;
+
+  @ApiProperty({
+    description: "Number of tree species for this period",
+    type: Number
+  })
+  treeSpeciesAmount: number;
+
+  @ApiProperty({
+    description: "Percentage of tree species for this period",
+    type: Number
+  })
+  treeSpeciesPercentage: number;
```

Comment:
🎨 It's a little weird to me to provide an amount and a percentage. Not a huge deal so leave it alone if you want, but it would make more sense to me to include the amount and the goal and let the FE calculate percentage when it needs to. Then if the goal is ever desired in the future, it's already included in the response data.